app/controllers/games_controller.rb

```diff
@@ -22,7 +22,7 @@ def update
 
   def show
     @game = Game.includes(:pieces).find(params[:id])
-    @piece = Piece.find(params[:id])
+    #@piece = Piece.find(params[:id])
   end
```

Comment:
rake db:drop
rake db:create

On Sun, Jul 24, 2016 at 6:59 PM, Aleks Real <notifications@github.com>
wrote:

> In app/controllers/games_controller.rb
> <https://github.com/Methodologists/metchess/pull/41#discussion_r71998548>:
>
> > @@ -22,7 +22,7 @@ def update
> >
> >    def show
> >      @game = Game.includes(:pieces).find(params[:id])
> > -    @piece = Piece.find(params[:id])
> > +    #@piece = Piece.find(params[:id])
> >    end
>
> @ruishine <https://github.com/ruishine> what do you mean by drop and
> recreate database? Because I tried rake db:migrate if that's what you
> meant...
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Methodologists/metchess/pull/41/files/8dd6edc51c1ed50a3375adc687dff608324a5e77#r71998548>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALOK7OxeByfkxCUbW7xfavos6nqbO-UVks5qY_x1gaJpZM4JTfyY>
> .
>



-- 
-Lin Rui Li